udata_hydra/db/resource.py

```diff
@@ -48,8 +48,9 @@ async def insert(
                         url = $3,
                         deleted = FALSE,
                         status = $4,
-                        priority = $5;"""
-            await connection.execute(q, dataset_id, resource_id, url, status, priority)
+                        priority = $5
+                    RETURNING *;"""
+            await connection.fetchrow(q, dataset_id, resource_id, url, status, priority)
```

Comment:
Good catch, indeed it's just a matter of consistency and reusability.
The idea if to be able to have consistent uses of CRUD methods, so that we can expect the same behaviour from any of those methods, if this one is used later.

I still wonder if the adding of RETURNING * could impact performances negatively, though?